slices/less.yaml

```diff
@@ -0,0 +1,32 @@
+package: less
+
+essential:
+  - less_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libtinfo6_libs
+    contents:
+      /usr/bin/less:
+      /usr/bin/lessecho:
+      /usr/bin/lesskey:
+
+  mime:
+    contents:
+      /usr/lib/mime/packages/less:
+
+  scripts:
+    essential:
+      - base-files_bin
+      - dash_bins
```

Comment:
We should omit `base-files_bin` & `dash_bins`, as they are not actually dependencies for the `less` package:
```
$ apt show less
...
Depends: libc6 (>= 2.34), libtinfo6 (>= 6)
...
```